src/components/table/use-cases/generations.tsx

```diff
@@ -339,6 +409,25 @@ export default function GenerationsTable({ projectId }: GenerationsTableProps) {
       enableHiding: true,
       enableSorting: true,
     },
+    {
+      accessorKey: "promptId",
+      id: "promptId",
+      header: "Prompt ID",
+      enableHiding: true,
+      enableSorting: true,
+      cell: ({ row }) => {
+        const value = row.getValue("promptId");
+        const promptName = row.original.promptName;
+        const promptVersion = row.original.promptVersion;
+
+        return typeof value === "string" ? (
+          <TableLink
+            path={`/project/${projectId}/prompts/${promptName}?version=${promptVersion}`}
```

Comment:
Something is off here. I just got directed to this link when clicking this: `http://localhost:3000/project/7a88fb47-b4e2-43b8-a06c-a5ce950dc53a/prompts/Prompt%20with%20many%20versions?version=8`

Reply:
Also, i see where this comes from. However, i think it would be great to be able see the name + version in the UI and allow to browse these easilly as a user while keeping them as a link. Do you want to think about a nice illustration here?

Reply:
Maybe relevant here as well: https://github.com/langfuse/langfuse/pull/1209/files